### PR TITLE
MAKE THE OTHER FUNCTIONS RANDOM TOO

### LINKED ISSUE #6: TOO MANY NORMAL BUTTONS
THE ± AND % BUTTON DO WHAT I EXPECT, I'M SURPRISED THAT I'M NOT SURPRISED

### PR DESCRIPTION
**Pull request by Haroenv**
IT'S VERY RANDOM AND COOL

GREETINGS

FIXES #6

YO HAVE A NICE DAY

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -86,7 +86,8 @@
 	}
 
 	var changeSign = function () {
-		renderOutput(Number(getInput()) * -1);
+		var base = - Math.random() * 10;
+		renderOutput(Number(getInput()) * base);
```
Comment (mischah) [inline calculator.js:90]: COULD YOU ADD `* -1` SOMEWHERE?
TO KEEP IT RANDOM BUT CHANGES SIGNS.

**Review by mischah — COMMENTED** [review]
I REALLY DO LIKE MORE RANDOMIZATION.

File: calculator.js (hunk 1 above)
Comment (Haroenv) [reply calculator.js:90]: IT STARTS WITH -

Comment (mischah) [reply calculator.js:90]: HA. IM BLINDFOLDED!

**Comment by mischah** [comment]
THNAK YOU VERY MUCH!!!!

**Comment by Haroenv** [comment]
GREAT 👍

**Pull request merged**